### PR TITLE
Debug load event listener

### PR DESCRIPTION
**Pull request by ninabrissey**
* A reference to a related issue in your repository - no issue
* A description of the changes proposed in the pull request - Debugged load event listener.
* @mentions of the person or team responsible for reviewing proposed changes - @jhortizu01 and @ninabrissey will review.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -112,7 +112,8 @@ var currentPoster = {
 //currentPoster = new Poster('', '', '')
 
 // event listeners go here 👇
-window.addEventListener('load', makeRandomPoster) {
+// window.addEventListener('load', makeRandomPoster);
+window.onload(makeRandomPoster);
```
Comment (jhortizu01) [inline src/main.js:116]: Great work!

File: src/main.js [hunk 2]
```diff
@@ -125,12 +126,11 @@ return Math.floor(Math.random() * array.length);
 };
 
 function makeRandomPoster() {
-  posterImage.src = images[getRandomIndex(images)]
-  posterTitle.innerText = titles[getRandomIndex(titles)]
-  posterQuote.innerText = quotes(getRandomIndex(quotes)]
+  posterImage.src = images[getRandomIndex(images)];
+  posterTitle.innerText = titles[getRandomIndex(titles)];
+  posterQuote.innerText = quotes[getRandomIndex(quotes)];
```
Comment (jhortizu01) [inline src/main.js:131]: Great job reviewing developer console for syntax errors.

File: src/main.js (hunk 1 above)
Comment (ninabrissey) [reply src/main.js:116]: Looks like we both missed that we did not uncomment line 115 and then remove 116.

Comment (ninabrissey) [reply src/main.js:116]: We will resolve this on the next push.

**Pull request merged**